common/prettify.pm

```diff
@@ -10,6 +10,7 @@ use warnings;
 
 use FileHandle;
 use Cwd;
+use DateTime::Format::Strptime;
```

Comment:
Is there something in standard modules that can do this?
https://perldoc.perl.org/modules#Standard-Modules